cg-spec/editors_draft.html

```diff
@@ -2660,9 +2659,118 @@ <h2>Other uses of the |unstar| mapping</h2>
 
       </section>
 
+      <section>
+        <h2>Referential opacity</h2>
+
+        <p>The way the |unstar| mapping is defined, the denotation of an embedded triple does not only depend on the denotation of its component terms, but also on their syntactical form (for IRIs and literals). This behavior is referred to as <a href="https://en.wikipedia.org/wiki/Opaque_context">referential opacity</a>. A consequence is that two different triples, that would be semantically equivalent if asserted, are not automatically considered to denote the same thing when embedded. In the example below, assuming <a data-cite="RDF11-MT#datatype-entailment">D-entailment</a>: while `"42"^^xsd:integer` and `"042"^^xsd:integer` denote the same thing (namely, the number 42), the two embedded triples are allowed to have different denotations, and so the entailment does not hold.</p>
```

Comment:
Instead of |unstar|, this should probably be a reference to a defined term _unstar mapping_, which can be defined in section 6.1. Otherwise, the notion of unstar mapping is never really defined.

Reply:
Good idea. Done in 5930b88